api/src/main/java/org/geysermc/geyser/api/extension/ExtensionManager.java

```diff
@@ -41,9 +41,19 @@ public abstract class ExtensionManager {
      * @param name the name of the extension
      * @return an extension with the given name
      */
+    @Deprecated
```

Comment:
RFC: do we want to deprecate getting an extension by name (instead of uuid)? Could have both without much issues

Reply:
yes, and add a `@deprecated` jd to the end of the jd explaining to use the other method

Reply:
coolio, added it